src/org/labkey/targetedms/chromlib/LibMolecule.java

```diff
@@ -3,30 +3,30 @@
 public class LibMolecule extends AbstractLibMolecule<LibMoleculePrecursor>
 {
     private Integer _moleculeListId;
-    private String _ionFormula;
-    private String _customIonName;
+    private String _chemicalFormula;
+    private String _moleculeName;
```

Comment:
does this need to be renamed as fragmentName?

Reply:
Brian requested that we keep the molecule one as MoleculeName and call the MoleculeTransition one as FragmentName, which I think is the case as currently coded.